src/App.js

```diff
@@ -25,33 +25,41 @@ class App extends Component {
     super();
 
     this.state = {
+      //the items is set as an array of active items on the screen
       items: [],
       points: 0,
     };
 
     // Uncomment this to spawn a single test item
-    //const testItem = this.spawnItem(Date.now());
-    //this.state.items.push(testItem);
+    // const testItem = this.spawnItem(Date.now());
+    // this.state.items.push(testItem);
 
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
-
-    console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in!
+  onItemClicked = (point) => {
+    let previousPoint = this.state.points
+    this.setState ({
+      points: previousPoint += point,
+    });
```

Comment:
the `+=` operator increments by adding and re-assigning the value. Actually, in this case, you just need to add! So `previousPoint + point` works perfectly well in this case.

Since you don't need to re-assign code, it may be better to make `previousPoint` a `const` variable, so:

```javascript
const previousPoint = this.state.points;
this.setState( {
  points: previousPoint + point
});

Also, you have a semicolon missing :)